packages/terra-arrange/src/terra-dev-site/doc/arrange/Arrange.1.doc.mdx

```diff
@@ -1,3 +1,14 @@
+import { Badge } from '../../../../package.json?dev-site-package';
```

Comment:
This import style should work and you should be able to change the rest of the imports this way as well.
```suggestion
import { Badge } from 'terra-arrange/package.json?dev-site-package';
```